src/gui/attributetable/qgsattributetablemodel.cpp

```diff
@@ -40,6 +41,7 @@ QgsAttributeTableModel::QgsAttributeTableModel( QgsVectorLayerCache *layerCache,
     , mLayerCache( layerCache )
     , mFieldCount( 0 )
     , mCachedField( -1 )
+    , mLoadWorker( NULL )
```

Comment:
That would be easy to`#define` just like `override`. But it has the advantage that on new compilers it can't be compared to integers by accident. While NULL doesn't offer any advantages over plain 0 nowhere.